README.md

```diff
@@ -12,8 +12,9 @@ Currently, the functionality of **connectivity_check** is also bundled in the ag
 
 ### Flags
 
-* *--host*:  The inventory host as dns name or ip address. Default is "api.openshift.com".
-* *--port*: The inventory port number. Default is 80.
+* *--host*:  **Deprecated, use *--url* instead.** The inventory host as dns name or ip address. Default is "api.openshift.com".
+* *--port*: **Deprecated, use *--url* instead.** The inventory port number. Default is 80.
```

Comment:
probably not a good default as it will never work by default? (i think api.openshift.com doesnt respond on 80?)

Reply:
It is deprecated, will be removed soon.